test/jest/roktManager.spec.ts

```diff
@@ -0,0 +1,115 @@
+import RoktManager, { IRoktLauncher, IRoktSelectPlacementsOptions } from "../../src/roktManager";
+
+describe('RoktManager', () => {
+    let roktManager: RoktManager;
+
+    beforeEach(() => {
+        roktManager = new RoktManager();
+    });
+
+    describe('constructor', () => {
+        it('should be initialized', () => {
+            expect(roktManager).toBeDefined();
+        });
+
+        it('should have a null launcher', () => {
+            expect(roktManager['launcher']).toBeNull();
+        });
+    });
+
+    describe('#attachLauncher', () => {
+        it('should attach a launcher', () => {
+            const launcher = {} as IRoktLauncher;
+            roktManager.attachLauncher(launcher);
+            expect(roktManager['launcher']).not.toBeNull();
+        });
+
+        it('should process the message queue if a launcher is attached', () => {
+            const launcher = {
+                selectPlacements: jest.fn()
+            } as unknown as IRoktLauncher;
```

Comment:
Can this just be
```suggestion
            const launcher: IRoktLauncher = {
                selectPlacements: jest.fn()
            };
```

or does that cause issues somewhere?

Reply:
Good catch. I miseed that one.